draft-ietf-quic-transport.md

```diff
@@ -933,7 +933,10 @@ proceeds with the handshake ({{handshake}}).  This commits the server to the
 version that the client selected.
 
 When the client receives a Version Negotiation packet from the server, it should
-select an acceptable protocol version.  If the server lists an acceptable
+select an acceptable protocol version. If the packet contains the version that
+the client initially offered, the client MUST terminate the connection
+using a QUIC_INVALID_VERSION_NEGOTIATION_PACKET error.
```

Comment:
It's not currently possible to send that error code.  (See #467.)

It might be better to just consider QUIC to have failed in this case.